### DISCUSSION TITLE
New user, Web UI seems to be missing CSS (Windows Local)

### DISCUSSION THREAD

**Discussion by Stone-D** [lllyasviel/Fooocus#2924] [Q&A] [answered]
I'm having a weird UI issue that I haven't been able to find elsewhere.

Windows 10 22H2, Microsoft Edge.

Doubleclicking either run.bat or run_realistic.bat successfully launches the browser tab and I can generate images (nothing updates live), but the UI is massive and extremely basic. It's as if there is no CSS.

![01 Top](https://github.com/lllyasviel/Fooocus/assets/522690/b02abdb8-a646-497b-9fd6-d7459623cea0)
![02 20%](https://github.com/lllyasviel/Fooocus/assets/522690/d4341878-5d93-4256-bd07-7087c2de3862)
![03 Bottom of textbox](https://github.com/lllyasviel/Fooocus/assets/522690/880e4dbc-e4a2-48e9-9d0f-d3d609d45dfa)
![04 Bottom](https://github.com/lllyasviel/Fooocus/assets/522690/9b85b06f-88f5-45e5-b8f4-f5d76b48357f)
![05 25% Zoom](https://github.com/lllyasviel/Fooocus/assets/522690/3c706175-df32-4fea-92d7-7a8850ed8953)

Any idea what I'm doing wrong?

**Comment by ILikeToasters** [comment, marked as answer]
I would check things like security settings since I just tested on Edge and worked fine. I normally use Brave browser.

**Reply by mashb1t** [reply]
@Stone-D make sure you don't use adblockers and/or Noscript extensions as well as check if anything is blocked in the network tab of your browser.
